app/views/registrations/school_profile/show.html.erb

```diff
@@ -1,13 +1,19 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: registrations_question_two_path) %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">What is your school Unique Reference Number(URN)?</h2>
     <%= form_with(model: @school_profile_form, url: registrations_school_profile_path) do |f| %>
       <%= f.govuk_error_summary %>
-      <label class="govuk-hint">
-        This is a 6 digit number. Not sure? <a target='_blank' href='https://www.get-information-schools.service.gov.uk/'>Search for your school</a> on our register.
-        <%= f.govuk_text_field(:urn, width: 20, label:  nil, link_errors: true) %>
-      </label>
+      <%= f.govuk_text_field(
+            :urn,
+            width: 20,
+            label: -> { tag.h1("What is your school Unique Reference Number (URN)?", class: "govuk-heading-l") },
```

Comment:
Did you mention on another PR that we can specify tag separately?

Reply:
Yeh, do as I say not as I do 😄